lib/http/HttpServer.ts

```diff
@@ -67,7 +67,10 @@ class HttpServer {
       res.writeHead(200, { 'Content-Type': 'application/json' });
       res.end(JSON.stringify(resJson));
     } catch (err) {
-      if (err.code === errorCodes.INVALID_ARGUMENT) {
+      if (
+        err.code === errorCodes.INVALID_ARGUMENT ||
+        err.code === swapErrorCodes.INVALID_RESOLVE_REQUEST
```

Comment:
I'm thinking I should've made this a `switch` statement when I wrote this originally, but now I think it's even more clearly the nicer choice. Would you mind rewriting these if/elses as a switch?

Reply:
Rewrote this as switch - looks much better now, thanks.